src/provisioner/provisioner.py

```diff
@@ -274,6 +313,27 @@ def cluster_bootstrap(cortx_conf_url: str, force_override: bool = False):
             cortx_conf, node_id, apply_phase, ProvisionerStatus.SUCCESS.value)
         Log.info(f"Finished cluster bootstrap on {node_id}:{node_name}")
 
+    @staticmethod
+    def Prepare_upgrade(_conf_idx: str, _tmp_idx: str):
```

Comment:
Please follow naming convention for functions in lower case letters: prepare_upgrade.
Also please check if you can name the function specific to  what it does like prepare_delta, get_delta,  or any apt name you may come up with.

Reply:
done